frontend/src/components/shared/Dropdown/Selectbox.tsx

```diff
@@ -204,8 +204,8 @@ class Selectbox extends React.PureComponent<Props, State> {
                 // Baseweb requires long-hand props, short-hand leads to weird bugs & warnings.
                 paddingRight: ".5rem",
                 paddingLeft: ".5rem",
-                paddingBottom: ".5rem",
-                paddingTop: ".5rem",
+                paddingBottom: ".4rem",
```

Comment:
There is also a difference in line height between `selectbox` (1.6rem) and `text_input` (1.4rem). Just setting line-height to `1.4rem` also seems to fix it for the selectbox. Are you sure that we want to keep the `1.6rem` line height here?

Reply:
@LukasMasuch Hmm, I tried updating the `lineHeight` for the `Root` override, but it made no difference to me... However, I tried it on the browser and it worked, so maybe I'm updating the wrong override? I'd prefer to keep the `padding` as it was and have it fixed by only updating the `lineHeight` as you suggested, but it didn't work for me. Were you able to make it work?

Reply:
Ok, updating it on both `Root` and `Input` worked! See [337f2c3](https://github.com/streamlit/streamlit/pull/4995/commits/337f2c3f3e0618b3dcc84d8bbd513838e9b4372c)